Anchor 0.30.1 build fails at pyth-solana-receiver-sdk 0.3.0 version

I'm using Anchor 0.30.1 version. I installed pyth-solana-receiver-sdk 0.3.0 version but build fails with this kind of errors.
```
error[E0599]: no function or associated item named `get_full_path` found for struct `MerklePriceUpdate` in the current scope
error[E0599]: no function or associated item named `create_type` found for struct `MerklePriceUpdate` in the current scope
error[E0599]: no function or associated item named `insert_types` found for struct `MerklePriceUpdate` in the current scope
```
Here is the full [error log](https://github.com/user-attachments/files/16139572/error.log).

--- a/target_chains/solana/pyth_solana_receiver_sdk/Cargo.toml
+++ b/target_chains/solana/pyth_solana_receiver_sdk/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "pyth-solana-receiver-sdk"
-version = "0.3.0"
+version = "0.3.1"
 description = "SDK for the Pyth Solana Receiver program"
 authors = ["Pyth Data Association"]
 repository = "https://github.com/pyth-network/pyth-crosschain"

--- a/target_chains/solana/pyth_solana_receiver_sdk/src/lib.rs
+++ b/target_chains/solana/pyth_solana_receiver_sdk/src/lib.rs
@@ -1,8 +1,17 @@
-use borsh::{BorshDeserialize, BorshSerialize};
-use pythnet_sdk::wire::v1::MerklePriceUpdate;
 use {
-    anchor_lang::{declare_id, prelude::*},
-    solana_program::{pubkey, pubkey::Pubkey},
+    anchor_lang::{
+        declare_id,
+        prelude::*,
+    },
+    borsh::{
+        BorshDeserialize,
+        BorshSerialize,
+    },
+    pythnet_sdk::wire::v1::MerklePriceUpdate,
+    solana_program::{
+        pubkey,
+        pubkey::Pubkey,
+    },
 };
 
 pub mod config;
@@ -19,12 +28,12 @@ pub const PYTH_PUSH_ORACLE_ID: Pubkey = pubkey!("pythWSnswVUd12oZpeFP8e9CVaEqJg2
 #[derive(Debug, BorshSerialize, BorshDeserialize, Clone)]
 pub struct PostUpdateParams {
     pub merkle_price_update: MerklePriceUpdate,
-    pub treasury_id: u8,
+    pub treasury_id:         u8,
 }
 
 #[derive(Debug, BorshSerialize, BorshDeserialize, Clone)]
 pub struct PostUpdateAtomicParams {
-    pub vaa: Vec<u8>,
+    pub vaa:                 Vec<u8>,
     pub merkle_price_update: MerklePriceUpdate,
-    pub treasury_id: u8,
+    pub treasury_id:         u8,
 }

--- a/target_chains/solana/pyth_solana_receiver_sdk/src/lib.rs
+++ b/target_chains/solana/pyth_solana_receiver_sdk/src/lib.rs
@@ -1,13 +1,8 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use pythnet_sdk::wire::v1::MerklePriceUpdate;
 use {
-    anchor_lang::{
-        declare_id,
-        prelude::*,
-    },
-    pythnet_sdk::wire::v1::MerklePriceUpdate,
-    solana_program::{
-        pubkey,
-        pubkey::Pubkey,
-    },
+    anchor_lang::{declare_id, prelude::*},
+    solana_program::{pubkey, pubkey::Pubkey},
 };
 
 pub mod config;
@@ -21,15 +16,15 @@ declare_id!("rec5EKMGg6MxZYaMdyBfgwp4d5rB9T1VQH5pJv5LtFJ");
 
 pub const PYTH_PUSH_ORACLE_ID: Pubkey = pubkey!("pythWSnswVUd12oZpeFP8e9CVaEqJg25g1Vtc2biRsT");
 
-#[derive(Debug, AnchorSerialize, AnchorDeserialize, Clone)]
-pub struct PostUpdateAtomicParams {
-    pub vaa:                 Vec<u8>,
-    pub merkle_price_update: MerklePriceUpdate,
-    pub treasury_id:         u8,
-}
-
-#[derive(Debug, AnchorSerialize, AnchorDeserialize, Clone)]
+#[derive(Debug, BorshSerialize, BorshDeserialize, Clone)]
 pub struct PostUpdateParams {
     pub merkle_price_update: MerklePriceUpdate,
-    pub treasury_id:         u8,
+    pub treasury_id: u8,
+}
+
+#[derive(Debug, BorshSerialize, BorshDeserialize, Clone)]
+pub struct PostUpdateAtomicParams {
+    pub vaa: Vec<u8>,
+    pub merkle_price_update: MerklePriceUpdate,
+    pub treasury_id: u8,
 }

